db/seeds.rb

```diff
@@ -5,3 +5,8 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+formal_educations = ["N Level", "O Level", "A Level", "NITEC", "Higher NITEC", "Diploma", "Degree", "Master", "PhD", "Other"]
```

Comment:
bring down the "Other", as this is already over the 120 character limit per line